File: Insert.py
import requests
def Insert():
	response=requests.post('https://checkoutstaff.000webhostapp.com/CTDT/Insert.php',data={'user':'hello','licno':'tn33e407','amount':'500'})
	#print(response.text)
	if "Succes" in response.text:
		print("Successfully Executed")
		#return 1
	else:
		print("Error in execution")
		#return 0
Insert()	

File: Insert.php
<?php
    $mysql_host="localhost";
    $mysql_user="";
    $mysql_pass="";
    $mysql_db="";
    $con=@mysqli_connect($mysql_host,$mysql_user,$mysql_pass) or die("Error");
    mysqli_select_db($con,$mysql_db);
    $user=$_POST['name'];
    $licno=$_POST['licno'];
    $amount=$_POST['amount'];
    $qry="Insert into User values('$user','$licno','$amount')";
    $r=mysqli_query($con,$qry);
    if($r)
    {
        echo "Successfully Inserted";
    }
    else
    {
        echo "Error";
    }
?>